pkg/booking/booking.go

```diff
@@ -57,11 +61,12 @@ func NewService(
 	}
 
 	return &Service{
-		client:       client,
-		logger:       logger,
-		cmAccount:    cmAccount,
-		transactOpts: transactOpts,
-		chainID:      chainID,
+		client:           client,
+		logger:           logger,
+		cmAccount:        cmAccount,
+		transactOpts:     transactOpts,
+		chainID:          chainID,
+		cmAccountAddress: &cmAccountAddr,
```

Comment:
I'd suggest to store addr as value, not as pointer - will be more convenient to use, since u'r passing it as value in func arg anyway